### PR TITLE
Add Android API to delete channel and channel group

### PR DESCRIPTION
**Pull request by dluksza**

### CODE REVIEW

File: lib/modules/notifications/AndroidNotifications.js [hunk 1]
```diff
@@ -91,4 +91,32 @@ export default class AndroidNotifications {
     }
     return Promise.resolve();
   }
+
+  deleteChannelGroup(groupId: string): Promise<void> {
+    if (Platform.OS === 'android') {
+      if (!(groupId instanceof string)) {
```
Comment (Salakar) [inline lib/modules/notifications/AndroidNotifications.js]: Shouldn't this be `if (typeof groupId !== 'string') {`, instanceof string would throw `string undefined`?

File: lib/modules/notifications/AndroidNotifications.js [hunk 2]
```diff
@@ -91,4 +91,32 @@ export default class AndroidNotifications {
...
+      return getNativeModule(this._notifications).deleteChannelGroup(
+        groupId
+      );
+    }
+    return Promise.resolve();
+  }
+
+  deleteChannel(channelId: string): Promise<void> {
+    if (Platform.OS === 'android') {
+      if (!(channnelId instanceof string)) {
```
Comment (Salakar) [inline lib/modules/notifications/AndroidNotifications.js]: Shouldn't this be `if (typeof channnelId !== 'string') {`, instanceof string would throw `string undefined`?

**Comment by chrisbianca** [comment]
@Salakar Other than the changes you've suggested, it looks good to me.  
Thanks @dluksza !!

**Comment by dluksza** [comment]
Updated the PR, applying suggestions from @Salakar.

BTW. what about my other PR https://github.com/invertase/react-native-firebase/pull/1074, will you find some time to look into it?

**Comment by chrisbianca** [comment]
Thanks for making those changes.  I am planning on looking through your other PR later today - it requires a bit more time :)

**Pull request merged**